src/_guides/language/effective-dart/style.md

```diff
@@ -98,11 +98,10 @@ extension SmartIterable<T> on Iterable<T> { ... }
 
 [extensions]: /guides/language/extension-methods
 
-### DO name libraries, packages, directories, and source files using `lowercase_with_underscores`. {#do-name-libraries-and-source-files-using-lowercase_with_underscores}
+### DO name packages, directories, and source files using `lowercase_with_underscores`. {#do-name-libraries-and-source-files-using-lowercase_with_underscores}
```

Comment:
It would be nice to introduce a new anchor here and leave a copy of the old one. Any ideas for a simpler one which covers the elements of this modified rule?

Reply:
> Any ideas for a simpler one which covers the elements of this modified rule?

Isn't lowercase_with_underscore called snake case? We could use that.
And just reading the first sentence of the section text, it seems like the summary term for packages, directories, and source files is "filenames".

> introduce a new anchor here and leave a copy of the old one

I'm not sure how to do that or what it would look like. Like this....?

```suggestion
### DO name packages, directories, and source files using `lowercase_with_underscores`.
{#do-snake-case-for-filenames}
{#do-name-libraries-and-source-files-using-lowercase_with_underscores}
```

Reply:
For leaving a copy of the old one, you can just add an anchor element with an ID before the header.

As for filenames, I'm not so sure, as a package isn't really a file name. We also don't call it snake_case anywhere on the Dart site, so being specific with `lowercase_with_underscore` is probably okay.

Giving it some more thought, I think it's just fine that it's long since people won't be typing it manually, so let's just make it something accurate like: `do-name-packages-and-file-system-entities-using-lowercase-with-underscores`

```suggestion
<a id="do-name-libraries-and-source-files-using-lowercase_with_underscores"></a>
### DO name packages, directories, and source files using `lowercase_with_underscores`. {#do-name-packages-and-file-system-entities-using-lowercase-with-underscores}
```

@MaryaBelanger How does that sound?